### DISCUSSION TITLE
1.1.3 not working for me, shuts down my Victron 150/35

### DISCUSSION THREAD

**Discussion by mfpvne** [softwarecrash/Victron2MQTT#8] [General]
I Installed 1.1.0 and it worked fine 4 me.
1.1.3 remotly shuts down my Victron 150/35, also not stable. i often got Error 500. After 1 Hour it was online again.
WifiAP is 1m away.

**Comment by softwarecrash** [comment]
Please try disconnecting the TX(ESP Side)-RX(victron Side) cable for test
